instrumentation/logback/logback-appender-1.0/javaagent/src/test/resources/logback-test.xml

```diff
@@ -12,10 +12,11 @@
   <logger name="abc" level="INFO"/>
   <logger name="def" level="WARN"/>
 
+  <logger name="DebugLogger" level="DEBUG"/>
+
   <root level="INFO">
     <appender-ref ref="console"/>
   </root>
 
-  <logger name="org.testcontainers" level="WARN"/>
```

Comment:
unrelated to this PR - there's no testcontainers logging in this module, not sure how this snuck in there originally